client/src/pages/Settings/CustomBooking/CustomBookingForm/CustomBooking.tsx

```diff
@@ -0,0 +1,220 @@
+import { LocalizationProvider, DesktopDatePicker } from '@mui/lab';
+import { Button, CircularProgress, InputLabel, Paper } from '@mui/material';
+import { Box } from '@mui/system';
+import { Formik, FormikHelpers } from 'formik';
+import FormInput from '../../../../components/FormInput/FormInput';
+import SettingHeader from '../../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../../interface/User';
+import { Profile } from '../../../../interface/Profile';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import { makeStyles, styled } from '@mui/styles';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+import editProfile from '../../../../helpers/APICalls/editProfile';
+import { useSnackBar } from '../../../../context/useSnackbarContext';
+import TableRow from '@mui/material/TableRow';
+import TableHead from '@mui/material/TableHead';
+import TableContainer from '@mui/material/TableContainer';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import ButtonBase from '@mui/material/ButtonBase';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import React from 'react';
+import useStyles from './makeStyles';
+import customBookingList from './CustomBooking';
+
+const Img = styled('img')({
+  margin: 'auto',
+  display: 'block',
+  maxWidth: '100%',
+  maxHeight: '100%',
+});
+
+const theme = createTheme({
+  typography: {
+    subtitle1: {
+      fontSize: 12,
+    },
+    body1: {
+      fontWeight: 500,
+    },
+    button: {
+      fontStyle: 'italic',
+    },
+  },
+});
+
+interface CustomBookingProps {
+  header: string;
+}
+
+const CustomBooking = ({ header }: CustomBookingProps): JSX.Element => {
+  const TAX_RATE = 0.0625;
+
+  function ccyFormat(num: number) {
+    return `${num.toFixed(2)}`;
+  }
```

Comment:
This works for now but for integration we should use Stripe's last 4 digits they provide for the customers payment method